Sources/WebUI/WebViewProxy.swift

```diff
@@ -81,7 +87,12 @@ public final class WebViewProxy: ObservableObject {
                 for await value in webView.publisher(for: \.canGoForward).bufferedValues() {
                     self?.canGoForward = value
                 }
-            }
+            },
+            Task { @MainActor [weak self] in
+                for await value in webView.scrollView.publisher(for: \.contentSize).bufferedValues() {
+                    self?._contentSize = value
+                }
+            },
         ]
```

Comment:
If the implementation of KVO is limited to iOS, as shown in the code below, some of the changes added in this PR might be redundant.
```suggestion
            },
        ]
        
        #if canImport(UIKit)
        tasks.append(
            Task { @MainActor [weak self] in
                for await value in webView.scrollView.publisher(for: \.contentSize).bufferedValues() {
                    self?._contentSize = value
                }
            }
        )
        #endif
```

Reply:
I also tried your proposed implementation once.

I am puzzled by the gap between the API implementation policy at the time of refreshable() and this implementation policy. I am concerned that if we proceed with the implementation strictly to be provided only on certain platforms, there will be a large number of canImport preprocessors in the source code in the future.

On the other hand, the proposed implementation is certainly leaner.

Reply:
I made a provisionally strict implementation in #56.
It is possible to make the strictness a bit looser as far as the compilation goes, but how much would be better?

Reply:
`WebView.refreshable()` is designed with strong consideration for `View.refreshable(action:)`. While `View.refreshable(action:)` is available on macOS, it does not function as expected. Therefore, I believe there is no need to modify the current implementation of `refreshable()` in WebUI.

On the other hand, `contentSize` is an attribute that is inherently expected to be usable on macOS as well. Since it is not currently provided, I believe this exceptional case should be explicitly expressed in the code responsible for implementing `contentSize`.

Reply:
I'll try to fix it according to your suggested approach.

Reply:
I did. [704e241](https://github.com/cybozu/WebUI/pull/54/commits/704e241f33f4a24c3fa5585a94d5aa649c160e58)